app/src/main/java/com/example/android_receipt_app/SecondFragment.kt

```diff
@@ -46,11 +47,14 @@ class SecondFragment : Fragment() {
         val title = binding.Title1.text.toString()
         val image = "https://upload.wikimedia.org/wikipedia/commons/b/bb/Carmen_Electra_2013.jpg"
         val description = binding.Description1.text.toString()
+        val creationDate = binding.Description1.text.toString()
```

Comment:
5. Part 2/2:

We want to have real date of adding receipt here.
To achieve it, instead of making creationDate exactly the same as description here -> make something like
**val creationDate = Date()**  
This code should save current date (so date and time of adding new receipt) in creationDate itself.